definitions/marts/looker_studio_marts/ecf/ls_ecf_provider_activity_partnership_changes.sqlx

```diff
@@ -0,0 +1,69 @@
+config {
+    type: "table",
+    bigquery: {
+        partitionBy: "event_date",
+        clusterBy: ["cpd_lead_provider_name"]
+    },
+    description: "This mart contains successful (200 status code) API requests made by a Lead Provider on the API path containing 'partnerships/ecf' to change a partnership's associated delivery partner. \n \
+	This mart includes changes where the occurred_at date is in the last 180 days only. \n \

```

Comment:
Do we need to make this one 180 days, if the main mart bringing things together is 180 days? Could we simply have the delta provided and if in future we want to filter to 180 days we can?

Reply:
The mart is also loaded into the dashboard. To avoid having to load a large dataset to then filter down the filtering has been applied in the mart itself. If you wish I can remove the date filter here and apply it in the dashboard.

Reply:
Fair enough! Makes sense to me.